bin/omarchy-launch-browser

```diff
@@ -1,3 +1,19 @@
 #!/bin/bash
 
-exec setsid uwsm app -- $(sed -n 's/^Exec=\([^ ]*\).*/\1/p' {~/.local,~/.nix-profile,/usr}/share/applications/$(xdg-settings get default-web-browser) 2>/dev/null | head -1) ${args[@]} $@
+browser_exec=$(sed -n 's/^Exec=\([^ ]*\).*/\1/p' {~/.local,~/.nix-profile,/usr}/share/applications/$(xdg-settings get default-web-browser) 2>/dev/null | head -1)
+
+case "$browser_exec" in
+*firefox*)
```

Comment:
Should expand to include all the firefox-based browsers we also cover in windows.conf: windowrule = tag +firefox-based-browser, class:(Firefox|zen|librewolf)